api/references/contribution-points.md

```diff
@@ -43,6 +44,23 @@ MetaDescription: To extend Visual Studio Code, your extension (plug-in) declares
 - [`viewsWelcome`](/api/references/contribution-points#contributes.viewsWelcome)
 - [`walkthroughs`](/api/references/contribution-points#contributes.walkthroughs)
 
+## contributes.authentication
+
+Contributes an authentication provider. This will set up an activation event for your provider and display it on in your extension's features.
```

Comment:
on or in?

Reply:
```suggestion
Contributes an authentication provider. This will set up an activation event for your provider and display it in your extension's features.
```